runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java

```diff
@@ -125,15 +128,15 @@ public class WorldMapPlugin extends Plugin
 
 		NOT_STARTED_ICON = new BufferedImage(questIconBufferSize, questIconBufferSize, BufferedImage.TYPE_INT_ARGB);
 		final BufferedImage notStartedIcon = ImageUtil.loadImageResource(WorldMapPlugin.class, "quest_not_started_icon.png");
-		NOT_STARTED_ICON.getGraphics().drawImage(notStartedIcon, 4, 4, null);
+		NOT_STARTED_ICON.createGraphics().drawImage(notStartedIcon, questIconOffset, questIconOffset, null);
```

Comment:
Why is `getGraphics()` changed to `createGraphics()` for these updated lines?

Reply:
They look to be the same 
![image](https://github.com/runelite/runelite/assets/309739/9de20e5b-6092-4fce-9785-65e1648d3d5f)